apis/sgv2-graphqlapi/src/main/java/io/stargate/sgv2/graphql/web/resources/StargateGraphqlContext.java

```diff
@@ -21,27 +21,23 @@
 import io.stargate.bridge.proto.QueryOuterClass.Query;
 import io.stargate.bridge.proto.QueryOuterClass.QueryParameters;
 import io.stargate.bridge.proto.QueryOuterClass.Response;
-import io.stargate.sgv2.common.grpc.StargateBridgeClient;
+import io.stargate.sgv2.api.common.grpc.StargateBridgeClient;
 import io.stargate.sgv2.graphql.schema.CassandraFetcher;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 import java.util.concurrent.atomic.AtomicReference;
-import javax.servlet.http.HttpServletRequest;
 
 public class StargateGraphqlContext {
```

Comment:
This context object is passed to the GraphQL engine (see `input.context(...)` calls in `GraphqlResourceBase`), then it can be retrieved from fetchers with `DataFetchingEnvironment.getContext()`. So in a way it's GraphQL doing its own simplified version of injection.

Turning it into a bean would be feasible I guess, but then we'd need to make sure that every fetcher is a managed bean as well... Overall I don't think it's worth it.